ws.ts

```diff
@@ -32,6 +39,10 @@ export const ws_handler = (
         get_events_by_filter: func_GetEventsByFilter;
         write_regular_event: func_WriteRegularEvent;
         write_replaceable_event: func_WriteReplaceableEvent;
+        // invitation
+        get_invite_to_space_event: func_GetInviteToSpaceByID;
```

Comment:
What does it do?

Reply:
In the test, I use this to confirm that the event has indeed been added to the database.

Reply:
Used to determine if there is an invite event in the DB, and return an error if not.